soh/soh/Enhancements/randomizer/3drando/logic.cpp

```diff
@@ -559,10 +561,12 @@ namespace Logic {
     // IsAdult = Age == AGE_ADULT;
 
     CanBlastOrSmash = HasExplosives || CanUse(MEGATON_HAMMER);
+    CanUseSword     = CanUse(MASTER_SWORD) || CanUse(KOKIRI_SWORD) || CanUse(BIGGORON_SWORD);
```

Comment:
CanUseSword was made to be a simple start for replacing IsAdult when used to refer to the master sword + other adult items.